library/regul.lib

```diff
@@ -45,9 +45,9 @@ F3 "" 0 0 50 H V C CNN
 ALIAS LM7905 LM7912 79L05
 DRAW
 S -200 -150 200 150 0 1 0 N
-X VI VI -400 50 200 R 40 40 1 1 I
-X VO VO 400 50 200 L 40 40 1 1 w
-X GND GND 0 -250 100 U 40 30 1 1 I
+X IN 1 -400 50 200 R 40 40 1 1 I
```

Comment:
The negative regulators have sligthly different pin numbering. Check the datasheets.